src/kompendium/source.ts

```diff
@@ -87,12 +87,28 @@ const getStyle =
         };
     };
 
-async function getLinks(
+async function getSourceFileLinks(
     component: JsonDocsComponent
 ): Promise<JsonDocsSource[]> {
-    const linkTags = component.docsTags.filter((tag) => tag.name === 'link');
+    const deprecatedLinkTags = component.docsTags.filter(
+        (tag) => tag.name === 'link'
+    );
+    if (deprecatedLinkTags.length > 0) {
+        // eslint-disable-next-line no-console
+        console.warn(
+            'Using the @link tag to link source files for display alongside examples is deprecated. ' +
+                'Use @sourceFileLink instead.'
+        );
+    }
 
-    return Promise.all<JsonDocsSource>(linkTags.map(getLink(component)));
+    const linkTags = component.docsTags.filter(
+        (tag) => tag.name === 'sourceFileLink'
```

Comment:
Perhaps we should use `@sourceFile` rather than `@sourceFileLink`?

Reply:
Yeah I'm all for shorter names :smile:

Reply:
⚡